<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testIHMClimingProjectClass</class>
 <widget class="QMainWindow" name="testIHMClimingProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testIHMClimingProject</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Param">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Paramètres</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Jouer">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Jouer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connexion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Titre">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>RA Climbing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Created">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Crée et designer par TAFA Rakel, RABEHI Khalis, LAGREVOL Kilian et VALETTE Jules</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Copyright">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>©Copyright tous droits reservés</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_User">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Creation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Créer un Compte</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testIHMClimingProject.qrc"/>
 </resources>
 <connections/>
</ui>
